spark/src/test/java/io/spark/ddf/analytics/BinningHandlerTest.java

```diff
@@ -35,7 +35,7 @@ public void testBinning() throws DDFException {
     Assert.assertTrue(ddf1.getSchemaHandler().getColumn("month").getColumnClass() == ColumnClass.FACTOR);
     // {'[2,4]'=1, '(4,6]'=2, '(6,8]'=3}
     System.out.println(">>>>> get2,4) == " + ddf1.getSchemaHandler().getColumn("month").getOptionalFactor().getLevelMap().get("[2,4]"));
-	Assert.assertTrue(ddf1.getSchemaHandler().getColumn("month").getOptionalFactor().getLevelMap().get("[2,4]") == 2);
+	Assert.assertTrue(ddf1.getSchemaHandler().getColumn("month").getOptionalFactor().getLevelMap().get("[2,4]") == 1);
 
```

Comment:
Binning discretizes a continuous column into a categorical one. There are three ways of doing that: 1. create bins with equal size, 2. create bins with equal frequency, 3. create bins with custom size.